common/src/API.php

```diff
@@ -17,6 +17,24 @@ public function __construct($api_domain, $shop_id, $api_key) {
         $this->api_key = (string) $api_key;
     }
 
+    public function getReviews(?string $last_synced): ?\SimpleXMLElement {
+        $params = [
+            'id' => $this->shop_id,
+            'code' => $this->api_key,
+            'detailed' => true,
+            'last_synced' => $last_synced,
+        ];
+        $url = $this->buildURL('https://' . $this->api_domain . '/api/1.0/product_reviews.xml', $params);
+        $response = Requests::get($url);
+        if (isset($response->status_code) & ($response->status_code >= 200 && $response->status_code < 300)) {
```

Comment:
In this case, I don't have a status message from our API, and I am not sure of exactly how to do it. 
The current solution seems to do the job, but I have a deep feeling that you won't like it 😄

Reply:
I considered using `throw_for_status` and catch `Requests_Exception_HTTP` but it did not work as I expected it.

Reply:
`&` -> `&&`

Also you don't need the extra parentheses around the second pair of conditions.

~~~
if ($response->status_code && $response->status_code >= 200 && $response->status_code < 300) {
~~~